editor/src/components/filebrowser/fileitem.tsx

```diff
@@ -574,7 +558,16 @@ class FileBrowserItemInner extends React.PureComponent<
     })
   }
 
-  onMouseDown = () => {
+  onMouseDown = (e: React.MouseEvent) => {
+    if (e.button === 0) {
+      if (this.props.fileType !== 'ASSET_FILE' && this.props.fileType !== 'IMAGE_FILE') {
+        this.props.setSelected(this.props)
+        if (this.props.fileType != null && this.props.fileType !== 'DIRECTORY') {
+          this.props.dispatch([EditorActions.openCodeEditorFile(this.props.path, true)], 'everyone')
```

Comment:
Why on mouse down and not click? 
In real vscode you have to click on a file to open it, and I believe that is more standard than doing actions on mouse down.

Reply:
This would have been because that was the previous logic, where we were tending to use mousedown rather than click events as we found it to be far more reliable at the time (years ago). This probably would be better suited to a click event now though.

Reply:
the only difference is that this feels slightly slower (since mousedown happens before the click event)

Reply:
(moved to logic to `onClick`)